docs/aws-marketplace.md

```diff
@@ -0,0 +1,163 @@
+iam:
+  withOIDC: true
+  serviceAccounts:
+  - metadata:
+      name: wego-service-account # Altering this will require a corresponding change in a later command
+      namespace: wego-system
+    roleOnly: true
+    attachPolicy:
+      Version: "2012-10-17"
+      Statement:
+      - Effect: Allow
+        Action:
+        - "aws-marketplace:RegisterUsage"
+        Resource: '*'
+
+# This section will create a single Managed nodegroup with one node.
+# Edit or remove as desired.
+managedNodeGroups:
+- name: ng1
+  instanceType: m5.large
+  desiredCapacity: 1
+```
+
+Create the cluster:
+
+```bash
+eksctl create cluster -f cluster-config.yaml
+```
+
+## [Optional] Step 3: Update your EKS cluster
+
+**If you created your cluster using the configuration file in Step 2, your cluster is
+already configured correctly and you can skip ahead to Step 4.**
+
+In order to use the Weave GitOps Core container product,
+your cluster must be configured to run containers with the correct IAM Policies.
+
+The recommended way to do this is via [IRSA](https://aws.amazon.com/blogs/opensource/introducing-fine-grained-iam-roles-service-accounts/).
+
+Use this `eksctl` configuration below (replacing the placeholder values) to:
+- Associate an OIDC provider
+- Create the required service account ARN
+
+Save the example below as `oidc-config.yaml`
+```
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+metadata:
+  name: CLUSTER_NAME # Change this
+  region: REGION # Change this
+
+# This section is required
+iam:
+  withOIDC: true
+  serviceAccounts:
+  - metadata:
+      name: wego-service-account # Altering this will require a corresponding change in a later command
+    roleOnly: true
+    attachPolicy:
+      Version: "2012-10-17"
+      Statement:
+      - Effect: Allow
+        Action:
+        - "aws-marketplace:RegisterUsage"
+        Resource: '*'
+
+```
+
+```bash
+eksctl utils associate-iam-oidc-provider -f oidc-config.yaml
+eksctl create iamserviceaccount -f oidc-config.yaml --approve
+```
+
+## Step 4: Fetch the Service Account Role ARN
+First retrieve the ARN of the IAM role which you created for the WeGo service account:
+
+```bash
+# replace the placeholder values with your configuration
+# if you changed the service account name from wego-service-account, update that in the command
+export SA_ARN=$(eksctl get iamserviceaccount --cluster <cluster-name> --region <region> | awk '/wego-service-account/ {print $3}')
+
+echo $SA_ARN
+# should return
+# arn:aws:iam::<account-id>:role/eksctl-<cluster-name>-addon-iamserviceaccount-xxx-Role1-1N41MLVQEWUOF
+```
+
+_This value will also be discoverable in your IAM console, and in the Outputs of the Cloud Formation
+template which created it._
+
+## Step 5: Install Weave GitOps
+
+Copy the Chart URL from your subscription page.
+
+```bash
+helm install wego <URL> \
+  --set serviceAccountRole="$SA_ARN"
+
+# if you changed the name of the service account
+helm install wego <URL> \
```

Comment:
```suggestion
Copy the Chart URL from your subscription page, or download the file to your workstation.

```bash
helm install wego <URL/PATH> \
  --set serviceAccountRole="$SA_ARN"

# if you changed the name of the service account
helm install wego <URL/PATH> \
```